packages/lesswrong/components/common/SearchBar.tsx

```diff
@@ -68,12 +68,14 @@ const styles = (theme: ThemeType): JssStyles => ({
     "&.open .ais-SearchBox-input": {
       display:"inline-block",
     },
+    "&.open .SearchBar-searchIcon": {
+      position: 'fixed',
+    },
   },
   searchInputAreaSmall: isEAForum ? {
     minWidth: 34,
   } : {},
   searchIcon: {
-    position: 'fixed',
```

Comment:
Without this change, the magnifying glass was staying fixed to the top when the header moved. I'm not sure why this isn't a problem for the standard header. But we only actually care about it being fixed while the search input is open so I think this change is ok.